presto-docs/src/main/sphinx/release/release-0.287.rst

```diff
@@ -0,0 +1,98 @@
+* Remove the configuration property ``use-legacy-scheduler`` and the corresponding session property ``use_legacy_scheduler``.   The property previously defaulted to true, and the new scheduler, which was intended to replace it eventually, was never productionized and is no longer needed. The configuration property ``max-stage-retries`` and the session property ``max_stage_retries`` have also been removed. :pr:`21952`
+* Add JSON as a supported output format in the Presto CLI. :pr:`22181`
+* Upgrade Alluxio to 310. :pr:`22012`
+* Upgrade Alluxio to 312. :pr:`22452`
+* Upgrade Airlift version to 0.209. :pr:`21943`
+* Add documentation for supported data `Type mapping <../connector/iceberg.html#type-mapping>`_  in the Iceberg connector. :pr:`22093`
+* Add usage documentation for :doc:`/clients/presto-cli`. :pr:`22265`
+* Add usage documentation for :doc:`/clients/presto-console`. :pr:`22349`
+* Add :doc:`Prestissimo Developer Guide <../prestissimo>` topic to the Presto documentation. :pr:`21953`
+
+Security Changes
+________________
+* Remove logback 1.2.3. :pr:`21819`
+* Add session property ``default-view-security-mode`` to choose the default security mode for view creation. :pr:`21956`
+
+Verifier Changes
+________________
+* Add support for extended bucket verification of INSERT and CTAS queries. This can be enabled by the configuration property ``extended-verification`` to verify each bucket's data checksum if the inserted table is bucketed. :pr:`22001`
+* Add support for extended partition verification of INSERT and CTAS queries. This can be enabled by the configuration property ``extended-verification`` to verify each partition's data checksum if the inserted table is partitioned. :pr:`21983`
+
+SPI Changes
+___________
+* Add replaceColumn method to com.facebook.common.Page. :pr:`22493`
+* Remove SPI method ConnectorMetadata.getTableLayouts() as deprecated. Add ConnectorMetadata.getTableLayoutForConstraint() as replacement. :pr:`21933`
+* Move `SortNode` to SPI module to be utilized in connector. :pr:`22497`
+
+Hive Changes
+____________
+* Fix a potential wrong results bug when footer stats are marked unreliable and partial aggregation pushdown is enabled.  Such queries will now fail with an error. :pr:`22011`
+* Improve the ``hive.orc.use-column-names`` configuration setting to no longer fail on reading ORC files without column names, but fall back to using Hive's schema. This change improves compatibility with legacy ORC files. :pr:`21391`
+* Add session property ``hive.dynamic_split_sizes_enabled`` to use dynamic split sizes based on data selected by query.  :pr:`22051`
+* Add support for Filelist caching for symlink tables.  :pr:`19145`
+* Add $row_id as a new hidden column. :pr:`22008`
+* Add system procedure ``system.invalidate_directory_list_cache()`` to invalidate directory list cache in Hive Catalog. :pr:`19821`
+
+Iceberg Changes
```

Comment:
```suggestion
Iceberg Connector Changes
```
Following the [Section naming](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#section-naming) guidelines in Release Notes Guidelines.